vagrant/provisioning/roles/minio/tasks/main.yml

```diff
@@ -0,0 +1,58 @@
+
+- name: create minio group
+  become: yes
+  group:
+    name: minio
+    state: present
+- name: create minio user
+  become: yes
+  user:
+    name: minio
+    group: minio
+    shell: /sbin/nologin
+    home: /home/minio
+- name: create minio bin directory
+  become: yes
+  file:
+    path: "{{root_folder}}/minio/{{item}}"
```

Comment:
Also please leave spaces when defining variables {{ variable }} instead of {{variable}}